assets/src/styles/campaigns/base/_mixins.scss

```diff
@@ -22,14 +22,17 @@
   transition: background $transition ease-in-out;
   padding: 0 40px;
   display: inline-block;
-  width: auto;
 
   &:not(:disabled):not(.disabled) {
     &:active {
       color: $foreground;
       background: $background;
     }
   }
+
+  &.article-load-more {
```

Comment:
looks good :+1: there's also `.btn-load-more-posts-click`button on Covers block that might appear on campaign pages